### PR TITLE
Storybook: Org Plugin Components

### PR DESCRIPTION
**Pull request by egnwd**
## Hey, I just made a Pull Request!

I added the plugin-org components to the storybook, as I was looking into them for #4519.

This is the list of components added:
image

I modified some of the components:
 - MembersList: some better spacing as they were bunched up to the line and not very wide at medium
 - Correct the size of the email icon in the GroupProfileCard as it was very small and not in keeping
 - Make the sizing of the grids uniform across the UserProfileCard and the GroupProfileCard (I couldn't see an obvious reason why they were different)

image
image

As shown below, I still need to look into the changeset.
Would really appreciate any feedback.

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)

### CODE REVIEW

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4520-k4yuyyy0srodfl41tlkosg7wgvwv2qin.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60288b55f285af803d8e7550

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4520-k4yuyyy0srodfl41tlkosg7wgvwv2qin.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60288b55f285af803d8e7550

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4520-k4yuyyy0srodfl41tlkosg7wgvwv2qin.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60288b55f285af803d8e7550

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4520-k4yuyyy0srodfl41tlkosg7wgvwv2qin.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60288b55f285af803d8e7550

**Comment by Fox32** [comment]
I wonder if this is a pattern we want to apply to more plugins. In contrast this duplicated the separate dev entrypoints ([`dev/index.tsx`](https://github.com/backstage/backstage/blob/master/plugins/org/dev/index.tsx)) in plugins (even though the org plugin isn't using it). Storybook is a bit nicer at it's easy to access at http://backstage.io/storybook while the dev entrypoints require a checkout. But the dev entrypoint makes the widget run in an environment that is much closer to a real Backstage app.

**Comment by egnwd** [comment]
@Fox32 Interesting point. I added it to the storybook as that's what was mentioned in the contributing guide and, after running the dev entrypoint and seeing it was empty, thought this might be the way to go. 

I feel like for some of the plugins that expose small components like this the storybook is quite nice, but for the plugins with a bit more to them the dev entrypoint is a lot more useful.

**Comment by benjdlambert** [comment]
Looks pretty good. you need to create a changeset which bumps the `@backstage/plugin-org` by `patch`.

You can find more info [here](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets)

File: plugins/org/src/components/Cards/User/UserProfileCard/UserProfileCard.tsx [hunk 1]
```diff
@@ -95,11 +95,11 @@ export const UserProfileCard = ({
   return (
     <InfoCard title={<CardTitle title={displayName} />} variant={variant}>
       <Grid container spacing={3} alignItems="flex-start">
-        <Grid item md={2} lg={1}>
+        <Grid item xs={12} sm={2} xl={1}>
           <Avatar displayName={displayName} picture={profile?.picture} />
         </Grid>
 
-        <Grid item md={10} lg={11}>
+        <Grid item md={10} xl={11}>
```
Comment (freben) [inline plugins/org/src/components/Cards/User/UserProfileCard/UserProfileCard.tsx:102]: odd that these don't line up with the breakpoints above - would have expected something like `xs={12} sm={10} xl={1}`

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4520-k4yuyyy0srodfl41tlkosg7wgvwv2qin.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60288b55f285af803d8e7550

File: plugins/org/src/components/Cards/User/UserProfileCard/UserProfileCard.tsx (hunk 1 above)
Comment (egnwd) [reply plugins/org/src/components/Cards/User/UserProfileCard/UserProfileCard.tsx:102]: Hmm, I'll take another look. Visually they looked fine but perhaps I missed something.

**Review by adamdmharvey — COMMENTED** [review]
Formatting is much improved 👍 💯 

Regarding storybook, I always think of it as reusable components that to be "things I can reuse in my plugin or app". While these are exported so they can be placed on entity cards inside of an app, are they generic enough that any other plugin might want to borrow them?  If we think the answer is no, I would have leaned only to the dev usage and a local `yarn start` personally. That's generally why the only storybook elements in the entire project today are all in `@backstage/core` (at least I thought).

**Comment by benjdlambert** [comment]
@adamdmharvey I guess it's super fine to have plugins also expose a storybook of their common patterns that other contributors will use when building on top of too? Maybe it would be better to make these more accessible. :)

**Comment by adamdmharvey** [comment]
@benjdlambert Absolutely!  It's the "common patterns that others will use" which is key, yeah?  Otherwise, storybook becomes just a demo showcase of existing plugins, which could just as easily be done via enhancements to their respective README pages with screen shots, etc. (and actually, a demo showcase of plugins that is more workable / usable / read might be a future fun enhancement to the plugins panel page 😁 )

**Comment by benjdlambert** [comment]
Exactly. I guess the difference is, is that unlike screenshots, they're living documentation. So they don't drift out of date so quickly, and can throw errors when things change that don't work anymore.
And yep, 100%. :shipit:

**Review by benjdlambert — APPROVED** [review]

**Pull request merged**